airflow/jobs/scheduler_job.py

```diff
@@ -1198,6 +1198,7 @@ def _create_dag_runs_dataset_triggered(
                     dag_hash=dag_hash,
                     creating_job_id=self.id,
                 )
+                Stats.incr("dataset.triggered_runs")
```

Comment:
```suggestion
                Stats.incr("dataset.triggered_dagruns")
```

Would rename for clarity